listenbrainz/listenstore/tests/util.py

```diff
@@ -15,12 +15,17 @@ def generate_data(test_user_id, user_name, from_ts, num_records):
     test_data = []
     artist_msid = str(uuid.uuid4())
 
-    for i in range(num_records):
+    if (from_ts == None):  #check for playing now listens
```

Comment:
We generally don't do parantheses in if conditions in Python. :)

this could be better written as `if from_ts is None`